arrayfire/arith.py

```diff
@@ -156,15 +156,8 @@ def clamp(val, low, high):
     vdims = dim4_to_tuple(val.dims())
     vty = val.type()
 
-    if not is_low_array:
-        low_arr = constant_array(low, vdims[0], vdims[1], vdims[2], vdims[3], vty)
-    else:
-        low_arr = low.arr
-
-    if not is_high_array:
-        high_arr = constant_array(high, vdims[0], vdims[1], vdims[2], vdims[3], vty)
-    else:
-        high_arr = high.arr
+    low_arr = low.arr if is_low_array else constant_array(low, vdims[0], vdims[1], vdims[2], vdims[3], vty)
```

Comment:
revert

Reply:
Still, I don't see a point to revert these changes. These variables are just temp variables and they are not changed later anyhow. But we drop extra 7 lines of code.